### PR TITLE
Update the tutorial and quickstart parts of documentation

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

**Comment by flaviodsr** [comment]
The doc update looks good, but shouldn't this be merged only when there is a new release (which includes the extensions feature) ?

File: docs/quickstart.md [hunk 1]
```diff
@@ -45,42 +45,39 @@ You should get an output similar to this:
 🚢 FuseML installing...
 
 Configuration...
-  🧭  system_domain: 
+  🧭  system_domain:
+  🧭  extension_repository: https://raw.githubusercontent.com/fuseml/extensions/main/installer/
```
Comment (jsuchome) [inline docs/quickstart.md:49]: OK, this should not point to `main` in released versions...

Comment (jsuchome) [reply docs/quickstart.md:49]: ... but that's OK for `main` branch of docu repo. We just need to fix it when branching

File: docs/tutorials.md [hunk 2]
```diff
@@ -16,16 +16,22 @@ check first the [quick start](quickstart.md) section.
 Running this example requires MLflow and KFServing to be installed in the same cluster as FuseML. For your
 convenience, an MLflow tracking server is already deployed as part of the main FuseML installation.
```
Comment (stefannica) [inline docs/tutorials.md:17]: no longer true

File: docs/tutorials.md [hunk 3]
```diff
@@ -16,16 +16,22 @@ check first the [quick start](quickstart.md) section.
...
 
-For a quick KFServing installation, you can use the scripts already hosted in the fuseml GitHub repository.
-Running the following will install KFServing on your cluster, along with all its prerequisites (cert-manager,
-istio and knative): 
+For a quick MLFlow and KFServing installation, you can need to run installer with the appropriate values for `--extensions`
```
Comment (stefannica) [inline docs/tutorials.md:19]: can or need, not both

File: docs/tutorials.md [hunk 4]
```diff
@@ -50,7 +56,7 @@ Under the `codesets/mlflow` directory, there are some example MLflow projects. F
 
 **4.** Register the codeset
 
-Register the example code as a FuseML versioned codeset artifact:
+From now one, you start using *fuseml* command line tool. Register the example code as a FuseML versioned codeset artifact:
```
Comment (stefannica) [inline docs/tutorials.md:59]: typo: from now on

File: docs/tutorials.md [hunk 5]
```diff
@@ -72,17 +78,6 @@ You may optionally log into the Gitea UI using the URL, username and password pr
...
-To retrieve these values from your cluster setup and replace them in the workflow definition, run:
-
-```bash
-export ACCESS=$(kubectl get secret -n fuseml-workloads mlflow-minio -o json| jq -r '.["data"]["accesskey"]' | base64 -d)
-export SECRET=$(kubectl get secret -n fuseml-workloads mlflow-minio -o json| jq -r '.["data"]["secretkey"]' | base64 -d)
-sed -i -e "/AWS_ACCESS_KEY_ID/{N;s/value: [^ \t]*/value: $ACCESS/}" workflows/mlflow-e2e.yaml
-sed -i -e "/AWS_SECRET_ACCESS_KEY/{N;s/value: [^ \t]*/value: $SECRET/}" workflows/mlflow-e2e.yaml
-```
-
 Use the modified example workflow definition to create a workflow in FuseML:
```
Comment (stefannica) [inline docs/tutorials.md:81]: the example no longer needs to be "modified"

File: docs/tutorials.md (hunk 2 above)
Comment (jsuchome) [reply docs/tutorials.md:17]: ah, missed that one

File: docs/tutorials.md (hunk 3 above)
Comment (jsuchome) [reply docs/tutorials.md:19]: true

File: docs/tutorials.md (hunk 4 above)
Comment (jsuchome) [reply docs/tutorials.md:59]: right

File: docs/tutorials.md (hunk 5 above)
Comment (jsuchome) [reply docs/tutorials.md:81]: also true

**Review by stefannica — APPROVED** [review]

**Pull request merged**